rpm-package/jellyfin.spec

```
@@ -0,0 +1,135 @@
+%global         debug_package %{nil}
+# jellyfin commit to package
+%global         commit f8a720d3d8adbdb1f092a42e592dae37ba3f25bb
+%global         gittag v3.5.2-5
+%global         shortcommit %(c=%{commit}; echo ${c:0:7}) 
+# Taglib-sharp commit of the submodule since github archive doesn't include submodules
+%global         taglib_commit ee5ab21742b71fd1b87ee24895582327e9e04776
+%global         taglib_shortcommit %(c=%{taglib_commit}; echo ${c:0:7})
+
+Name:           jellyfin
+Version:        3.5.2.git%{shortcommit}
```

This is going to lead to packages that don't successively update their predecessor when the `shortcommit` value is not lexigraphically greater than the previous release's `shortcommit`.

The proper solution here is to not use the git _sha1_ for official releases along with https://github.com/jellyfin/jellyfin/pull/418#issuecomment-451666159.

---

Since i cannot use a `-` in the `Version:` tag and i see now that `sha1` is not predicable maybe we should  use something like `3.5.2.5` for git-tag `v3.5.2-5`?
```suggestion
Version:        3.5.2.5
```

---

Ok changed version to 10.0.0 which would work as long as no dashes are in the git tag.